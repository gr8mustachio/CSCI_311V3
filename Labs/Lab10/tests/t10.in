build
11
0   35
1   67
2   28
3   51
4   84
5   27
6   46
7   17
8   18
9   27
10  46
getSize
getCapacity
print
push 11 19
print
pop
print
push 12 27
push 13 34
push 14 19
print
pop
print
updatePriority 7 13
updatePriority 5 29
print
updatePriority 10 25
print
