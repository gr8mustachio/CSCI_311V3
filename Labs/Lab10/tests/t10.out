17 18 27 35 27 28 46 51 67 84 46 
7 8 5 0 9 2 6 3 1 4 10 
3 8 5 7 9 2 6 0 1 4 10 
Size of the heap is 11
Capacity of the heap is 11
17 18 27 35 27 28 46 51 67 84 46 
7 8 5 0 9 2 6 3 1 4 10 
3 8 5 7 9 2 6 0 1 4 10 

17 18 19 35 27 27 46 51 67 84 46 28 
7 8 11 0 9 5 6 3 1 4 10 2 
3 8 11 7 9 5 6 0 1 4 10 2 

18 27 19 35 28 27 46 51 67 84 46 
8 9 11 0 2 5 6 3 1 4 10 
3 8 4 7 9 5 6 -1 0 1 10 2 

18 27 19 35 28 27 19 51 67 84 46 27 34 46 
8 9 11 0 2 5 14 3 1 4 10 12 13 6 
3 8 4 7 9 5 13 -1 0 1 10 2 11 12 6 

19 27 19 35 28 27 46 51 67 84 46 27 34 
11 9 14 0 2 5 6 3 1 4 10 12 13 
3 8 4 7 9 5 6 -1 -1 1 10 0 11 12 2 

Warning: could not update priority for the item with key 7
Warning: could not update priority for the item with key 5
19 27 19 35 28 27 46 51 67 84 46 27 34 
11 9 14 0 2 5 6 3 1 4 10 12 13 
3 8 4 7 9 5 6 -1 -1 1 10 0 11 12 2 

19 25 19 35 27 27 46 51 67 84 28 27 34 
11 10 14 0 9 5 6 3 1 4 2 12 13 
3 8 10 7 9 5 6 -1 -1 4 1 0 11 12 2 

